FilterViewController.m

```diff
@@ -27,11 +27,11 @@ - (void)viewDidLoad {
     MHCollapsibleViewManager *labels = [[MHCollapsibleViewManager alloc] initManagerWithAnimation:UITableViewRowAnimationMiddle topHierarchyTitle:@"Labels" tableView:self.tableView];
     
     //sends double array for filternames and single array for header lines
-    [labels setDataWithFilterNames:@[self.returnLabelArray] headerTitles:@[@"Labels"]];
+    [labels setDataWithFilterNames:self.returnLabelArray headerTitles:@[@"Labels"]];
     
     MHCollapsibleViewManager *surveys = [[MHCollapsibleViewManager alloc] initManagerWithAnimation:UITableViewRowAnimationMiddle topHierarchyTitle:@"Surveys" tableView:self.tableView];
     
-    NSArray *surveyAnswers = @[@[self.returnSurveyArray], @[self.returnSurveyArray], @[self.returnSurveyArray]];
+    NSArray *surveyAnswers = @[self.returnSurveyArray, self.returnSurveyArray, self.returnSurveyArray];
```

Comment:
If you are looking to concatenate these arrays instead of having and array of 3 arrays you will need to do the following:
```
NSArray surveyAnswers = [self.returnSurveyArray arrayByAddingObjectsFromArray:self.returnSurveyArray];
surveyAnswers = [surveyAnswers arrayByAddingObjectsFromArray:self.returnSurveyArray];
```
OR
```
NSMutableArray surveyAnswers = [self.returnSurveyArray mutableCopy];
[surveyAnswers addObjectsFromArray:self.returnSurveyArray];
[surveyAnswers addObjectsFromArray:self.returnSurveyArray];
```
If you get issues from trying to concatenate the same array of the same objects multiple times you could try addObjectsFromArray:self.returnSurveyArray.copy instead. I've never tried concatenating the same array with itself before.